### DISCUSSION TITLE
Rancher returns failed to verify certificate: x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by bholexy** [argoproj/argo-cd#20564] [Q&A]
I added a remote cluster to ArgoCD but argoCD keeps giving me this error
Get "https://rancher.xxxxxxx.com/k8s/clusters/c-m-yyyy/version?timeout=32s": tls: failed to verify certificate: x509: certificate signed by unknown authority

I added ArgoCD cert to argocd-server-tls and argocd-repo-server-tls and also uploaded the the rancher certificate to Argocd TLS certificate upload UI

**Comment by jaxomccl-sel** [comment]
Were you able to find a solution to this @bholexy?

**Reply by bholexy** [reply]
Yes. I added the rancher root certificate to Argo CD
